peter_rockwood/HW1.md

```diff
@@ -0,0 +1,234 @@
+
+### Homework #1
+(Due Monday 1/12)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
```

Comment:
Strong work throughout! I only have a couple of edge cases in problem #1 that tripped you up--nice work!